coop_product_coefficient/models/product_template.py

```diff
@@ -262,13 +263,15 @@ def recompute_base_price(self):
     @api.multi
     def use_theoritical_price(self):
         for template in self:
-            template.list_price = template.theoritical_price
+            template.with_context(skipp_double_write=True).write({
```

Comment:
```suggestion
            template.with_context(skip_price_update=True).write({
```
Please name it like this